pkg/operator/ceph/cluster/osd/envs.go

```diff
@@ -49,6 +49,8 @@ const (
 	CrushDeviceClassVarName             = "ROOK_OSD_CRUSH_DEVICE_CLASS"
 	CrushInitialWeightVarName           = "ROOK_OSD_CRUSH_INITIAL_WEIGHT"
 	OSDStoreTypeVarName                 = "ROOK_OSD_STORE_TYPE"
+	ReplaceOSDIDVarName                 = "ROOK_REPLACE_OSD"
+	RelaceOSDPathVarName                = "ROOK_REPLACE_OSD_PATH"
```

Comment:
Is this env var still used? Or is the path just detected inside the osd prepare job?

Reply:
No. Its not used anymore. The path is detected inside prepare pod job using ceph-volume list. Removed it.